src/media/css/games/home.styl

```diff
@@ -0,0 +1,106 @@
+@import '../lib';
+
+
+.games-home {
+    display: flex;
+    height: 100%;
+    flex-wrap: wrap;
+    justify-content: center;
+    width: 100%;
+
+    > section {
+        padding: 30px 120px;
+        width: 100%;
+    }
+}
+
+.games-home-primary {
+    display: flex;
+    flex-direction: column;
+    height: 375px;
+    width: 100%;
+}
+
+.games-home-primary-main {
+    align-items: center;
+    background-size: cover;
+    display: flex;
+    flex-grow: 1;
+    justify-content: center;
+    position: relative;
+
+    .install {
+        border: 1px solid $greyscale-white;
+        border-radius: 2px;
+        font-size: 30px;
+        font-style: italic;
+        font-weight: 500;
+        padding: 17px 20px 45px;
+        z-index: 2;
+    }
+}
+
+.games-home-primary-metadata-container {
+    align-self: flex-end;
+    background: $action-success;
```

Comment:
Since this isn't a successful action, we should probably use an intermediate variable for this, and set that variable equal to `$color--action-success` or whatever that value is.